src/python/pants/option/subsystem_test.py

```diff
@@ -70,6 +72,33 @@ class GoodToGo(Subsystem):
         args=["./pants"],
         bootstrap_option_values=None,
     )
-    GoodToGo.register_options_on_scope(options)
+    GoodToGo.register_options_on_scope(options, UnionMembership({}))
 
     assert not caplog.records, "The current blessed means of registering options should never warn."
+
+
+def test_register_plugin_options() -> None:
+    class Electrical(Subsystem):
+        options_scope = "electrical"
+
+    class LampPlugin:
+        is_on = BoolOption(default=False, help="Luxo Jr.")
+
+    class Blender:
+        contents = StrListOption(help="brrrrr")
+
+    options = Options.create(
+        env={},
+        config=Config.load([]),
+        known_scope_infos=[Electrical.get_scope_info()],
+        args=["./pants"],
+        bootstrap_option_values=None,
+    )
+    Electrical.register_options_on_scope(
```

Comment:
Yes! This test doesn't use any `rule`s. So calling `register_plugin_options` wouldn't really _do_ anything because there's nowhere for the return value to go. :unamused:

Reply:
Although maybe we could do:

```python
UnionMembership.from_rules(*LampPlugin.register_plugin_options(), Blender.register_plugin_options())
```

I'd approve that PR.